cantabular/table.go

```diff
@@ -275,6 +482,32 @@ func decodeValues(ctx context.Context, dec jsonstream.Decoder, dims Dimensions,
 	return rowCount, nil
 }
 
+func decodeValuesJson(ctx context.Context, dec jsonstream.Decoder, dims Dimensions, w io.Writer) (getObsResponse GetObservationsResponse, err error) {
+
+	obsResp := json.NewEncoder(w)
+	var getObservationsResponse GetObservationsResponse
+
+	// Obtain the Json objects according to the cantabular dimensions and counts
+	for ti := dims.NewIterator(ctx); dec.More(); {
+		count, err := dec.DecodeNumber()
+		if err != nil {
+			return GetObservationsResponse{}, fmt.Errorf("error decoding count: %w", err)
+		}
+		row, err := ti.createJsonObject(dims, count.String())
+		if err != nil {
+			return GetObservationsResponse{}, fmt.Errorf("error parsing a csv row: %w", err)
+		}
+		obsResp.Encode(row)
+		getObservationsResponse.Observations = append(getObservationsResponse.Observations, row)
+
+		if err := ti.Next(); err != nil {
+			return GetObservationsResponse{}, fmt.Errorf("error iterating to next row: %w", err)
```

Comment:
Think this error should be json node not row?

Reply:
This has been updated

Reply:
Just this one to go.

Reply:
This should now be fixed